plugins/tiddlywiki/filesystem/filesystemadaptor.js

```diff
@@ -14,16 +14,23 @@ A sync adaptor module for synchronising with the local filesystem via node.js AP
 
 // Get a reference to the file system
 var fs = $tw.node ? require("fs") : null,
-	path = $tw.node ? require("path") : null;
+	path = $tw.node ? require("path") : null,
+    	WikiExport = require('$:/core/modules/wiki.js');
```

Comment:
Rather than reusing those three methods from wiki.js directly we should abstract them into a utility module.

Reply:
I would rather give the syncadaptor a reference to the syncer anyway, so I'll just remove this code again.